apex_ocr/roi.py

```diff
@@ -49,8 +51,11 @@
     "REV_RES_WIDTH": 60,
 }
 
+RESOLUTION_DIR = Path("/home/apex/apex-ocr-common/apex_ocr/data/")
+
```

Comment:
This will need to change to be platform-agnostic. Maybe a relative path outside of apex_ocr - doesn't this get saved out by the server?